typings/starlette/formparsers.pyi

```diff
@@ -30,7 +30,6 @@ class MultiPartMessage(enum.Enum):
     HEADERS_FINISHED: Literal["7"] = ...
     END: Literal["8"] = ...
 
-def _user_safe_decode(src: bytes, codec: str) -> str: ...
```

Comment:
did we add this manually before? curious why a private function got added to stub

Reply:
Because we uses this function